modules/features/referrals/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+plugins {
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.ksp)
+    alias(libs.plugins.hilt)
+    alias(libs.plugins.kotlin.parcelize)
+}
+
+android {
+    namespace = "au.com.shiftyjelly.pocketcasts.referrals"
+    buildFeatures {
+        buildConfig = true
+        compose = true
+    }
+}
+
+dependencies {
```

Comment:
There are some configuration issues here. If you run `./gradlew buildHealth` it will share advice with potential fixes.

```
Advice for :modules:features:profile
Unused dependencies which should be removed:
  implementation(libs.coroutines.reactive)

Advice for :modules:features:referrals
Existing dependencies which should be modified to be as indicated:
  implementation(libs.dagger.hilt.android) (was api)
  implementation(project(":modules:services:compose")) (was api)
  implementation(project(":modules:services:localization")) (was api)
  implementation(project(":modules:services:model")) (was api)
```

Reply:
`libs.coroutines.reactive` is used in the vm:

`import kotlinx.coroutines.reactive.asFlow`. 

Do we know why we're getting it as an issue?

Reply:
Because the `ReferralsViewModel` got moved from the `:profile` module to `:referrals` module. And now `asFlow()` extension is no longer used anywhere in the `:profile` module.

Reply:
🤦‍♀️ it was for profile module